### PR TITLE
102852 

### PR DESCRIPTION
**Pull request by jennyesoo**
add change request function

### CODE REVIEW

File: HP.Pulsar.Search.Keyword/DataReader/ChangeRequestReader.cs [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+        }
+
+        private string GetApproversCommandText()
+        {
+            return @"
+SELECT dcr.id as ChangeRequestId,
+    stuff((
+            SELECT ' { ' + e.Name + ' } '
+            FROM ActionApproval AS a WITH (NOLOCK)
+            INNER JOIN Employee AS e WITH (NOLOCK) ON a.ApproverId = e.Id
```
Comment (wooly905) [inline HP.Pulsar.Search.Keyword/DataReader/ChangeRequestReader.cs:99]: UserInfo

File: HP.Pulsar.Search.Keyword/DataReader/ChangeRequestReader.cs [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+JOIN ActionStatus AStatus ON AStatus.id = di.STATUS
+JOIN UserInfo ui ON ui.userid = di.OwnerID
+WHERE (
+        @ChangeRequestId = - 1
+        OR di.Id = @ChangeRequestId
+        )
+";
+        }
+
+        private string GetApproversCommandText()
```
Comment (wooly905) [inline HP.Pulsar.Search.Keyword/DataReader/ChangeRequestReader.cs:92]: Per discussion, improve this function.

**Review by wooly905 — COMMENTED** [review]
Improve the read function to make it faster.

**Comment by jennyesoo** [comment]
Optimize approver performance

Old version 42~45 s
use C# code(this version) 28~35 s (usually 30s)

**Pull request merged**